release-notes/10.0/preview/preview3/sdk.md

```diff
@@ -2,12 +2,78 @@
+
+To use the new scripts, you will need to run the `dotnet completions generate [SHELL]` command. This will generate a script that you can source in your shell. For example, to generate a script for `pwsh`, you could add the following to your $PROFILE$:
+
+```powershell
+dotnet completions script pwsh | out-String | Invoke-Expression -ErrorAction SilentlyContinue
+```
+
+This will generate a script that will be run every time you start a new PowerShell session. You can also run the command manually to generate the script and source it in your current session. Similar mechanisms can be used for the other supported shells!
+
+The overall completions-generation mechanisms are pluggable, so if you'd like to see new shells supported or enhancements made to your existing shell's scripts, raise an issue over at dotnet/sdk!
+
+## Containers
+
+### Console apps can natively create container images
+
+Console apps can now create container images via `dotnet publish /t:PublishContainer` without having to set the `<EnableSdkContainerSupport>` property to `true` in the project file. This brings them into line with the existing behavior for ASP.NET Core apps and Worker SDK apps, which have supported this for a while. If you need to disable this support for any reason, you can set the `<EnableSdkContainerSupport>` property to `false` in the project file.
+
+### Explicitly control the image format of containers
+
+The container images the .NET SDK produces today are either in a Docker-specific format or the more standardized OCI format. Which one our tooling uses is based on two things - if the container is a multi-architecture container, and the format of the base image being used. If the container is a multi-architecture container, we will always use the OCI format. Otherwise, for single-architecture containers, we will use the format of the base image. This means that any container generated using the Microsoft .NET base images will be in the Docker formats. Many users are moving to OCI images across their tooling, and so we have added a new property to the SDK, `<ContainerImageFormat>`, which allows you to explicitly set the format of the container image. This property can be set to `Docker` or `OCI`, and will override the default behavior. This property is only used when building a container image, and will not affect any other operations.
+
+## Testing
+
+### Support for Microsoft Testing Platform in `dotnet test`
+
+Starting with .NET 10 Preview 3, `dotnet test` now natively supports the [Microsoft.Testing.Platform](https://aka.ms/mtp-overview). You can enable this feature by adding the following configuration to your dotnet.config file (which itself is a [draft specification](https://github.com/dotnet/designs/pull/328) that we would love your feedback on):
 
-## Feature
+ ```toml
```

Comment:
Let's not brand as toml and brand as ini instead. This is what we currently use for parsing and the given snippet for p3 isn't valid toml anyways.

~~~suggestion
 ```toml
~~~